docs/demo.md

```diff
@@ -128,7 +128,7 @@ for project in projects:
   print(project["createdDateTime"])
 ```
 
-### Known Cadl Issues for Parity (Changlong)
+### Known Cadl Issues for Parity (Yuchao)
```

Comment:
Shall I take this part? If that, the flow is `yuchao`->`Isabella`->`Changlong` instead of `yuchao`->`Changlong`->`Isabella`->`Changlong` which can save time

Reply:
sure, I'm ok with that.